label_sync/cluster/label_sync_cron_job.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: label-sync-cron
+spec:
+  schedule: "0 */6 * * *"    # Every day 23:17 / 11:17PM
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    metadata:
+      labels:
+        app: label-sync
+    spec:
+      template:
+        spec:
+          containers:
+            - name: label-sync
+              image: gcr.io/k8s-testimages/label_sync:v20190126-6c4304780
+              args:
+              - --config=/etc/config/labels.yml
```

Comment:
I suggest we change labels.yml  into labels.yaml, since the example is named after .yaml and yaml is used as well. Be consistent in all the context.